Sources/SKCore/SourceKitLSPOptions.swift

```diff
@@ -200,6 +200,21 @@ public struct SourceKitLSPOptions: Sendable, Codable {
     self.workDoneProgressDebounce = workDoneProgressDebounce
   }
 
+  public init?(fromLSPAny lspAny: LSPAny?) throws {
```

Comment:
Should we move this type to LSP so that clients can also construct it (though for that to actually be true we probably need to split LSP out of this repo, but still... seems like a reasonable place for it to live)?

Reply:
I started moving it to the LanguageServerProtocol module and then decided against it. I can’t really tell why but it didn’t feel at home there.